index.bs

```diff
@@ -236,25 +237,25 @@ aligned (8) class AV1CodecConfigurationRecordv1 {
   unsigned int (1) marker = 1;
   unsigned int (3) version = 1;
   unsigned int (3) seq_profile;
+  unsigned int (1) still_picture;
```

Comment:
still_picture is IMO unnecessary. It just signals that the stream (supposedly) has one frame. It on itself is not really useful, and other boxes probably already tell demuxers that much. It can be removed, especially if it helps with byte alignment for other changes.

And going back to my suggestion to make the first byte contain only the marker, a bunch of reserved bits and the version, it would in this case also let the seq_profile and seq_level_idx_0 pair neatly fill a single byte.

Reply:
My idea for the first byte when suggesting that was basically being the "header" of AV1CodecConfigurationRecordv1, containing the marker and the version number. Second byte onwards would be the ones with version dependent init params.

Unlike the mandatory fields, it's mostly cosmetic.

Reply:
Ah, I didn't catch that you were suggesting widening version to handle the realignment. 
b704630 does that. It definitely looks better this way, IMO. Thanks!